custom_components/ynab/__init__.py

```diff
@@ -136,20 +136,43 @@ def __init__(self, hass, config):
             _LOGGER.debug(
                 "Recieved data for: uncleared transactions: %s", uncleared_transactions
             )
+            
+            total_balance = 0
+            # get account data
+            for a in self.get_data.accounts:
+                if a.on_budget:
+                  total_balance += a.balance
+                  
+            # get to be budgeted data
+            self.hass.data[DOMAIN_DATA]["total_balance"] = total_balance / 1000
+            _LOGGER.debug(
+                "Recieved data for: total balance: %s",
+                (total_balance),
```

Comment:
Please change this to self.hass.data[DOMAIN_DATA]["total_balance"] so that the logger outputs balance in the same  format as other values.

Reply:
Done as of the latest commit. Well spotted!